src/common/com/intellij/plugins/haxe/ide/generation/CreateGetterSetterFix.java

```diff
@@ -46,7 +42,7 @@ boolean accept(String name, Set<String> names) {
     }, GETTERSETTER {
       @Override
       boolean accept(String name, Set<String> names) {
-        return !names.contains(HaxePresentableUtil.getterName(name)) &&
+        return !names.contains(HaxePresentableUtil.getterName(name)) ||
```

Comment:
Ah, I see.  You're checking for each of them below when you actually insert the function.